src/components/modal/helpers/bv-modal-event.class.spec.js

```diff
@@ -36,6 +36,10 @@ describe('modal > BvModalEvent', () => {
   })
 
   it('supports cancelable events via deprecated evt.cancel()', async () => {
+    // Mock `console.warn()` to prevent a console warning and
+    // check wether a warning about the `evt.cancel()` call
+    // was made
+    console.warn = jest.fn()
```

Comment:
@tmorehouse Is this fine like this or do wee need to revert the mock after execution of the test in any way?

Reply:
Hmmm good question... might want to revert... unless this is the last test in the file.

Reply:
Did it just to be safe.